pkg/util/utils.go

```diff
@@ -301,36 +301,36 @@ func getTomlStorage(storeOptions *storage.StoreOptions) *tomlConfig {
 // for the volume API
 // It also returns the path where all named volumes will be created using the volume API
 func GetDefaultStoreOptions() (storage.StoreOptions, string, error) {
+	var (
+		defaultRootlessRunRoot   string
+		defaultRootlessGraphRoot string
+		err                      error
+	)
 	storageOpts := storage.DefaultStoreOptions
 	volumePath := "/var/lib/containers/storage"
+
 	if rootless.IsRootless() {
-		var err error
 		storageOpts, err = GetRootlessStorageOpts()
 		if err != nil {
 			return storageOpts, volumePath, err
 		}
+
 		volumePath, err = GetRootlessVolumeInfo()
 		if err != nil {
 			return storageOpts, volumePath, err
 		}
+	}
 
-		storageConf := StorageConfigFile()
-		if _, err := os.Stat(storageConf); err == nil {
-			defaultRootlessRunRoot := storageOpts.RunRoot
-			defaultRootlessGraphRoot := storageOpts.GraphRoot
-			storageOpts = storage.StoreOptions{}
-			storage.ReloadConfigurationFile(storageConf, &storageOpts)
+	storageConf := StorageConfigFile()
+	if _, err := os.Stat(storageConf); err == nil {
```

Comment:
the issue is that `err` exists only in the `if` scope.  if we change it to "_, err = ...` then it works fine

Reply:
Fixed